blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -146,7 +149,7 @@ const LargePromo = ({ customFields }) => {
         <div className="row">
           {customFields.showImage
           && (
-            <div className="col-sm-12 col-md-xl-6">
+            <div className="col-sm-12 col-md-xl-6 flex-col">
```

Comment:
was this css change deliberate?

Reply:
yes, it is used on other promo blocks and on other places like
https://github.com/WPMedia/fusion-news-theme-blocks/blob/92e2562e2917f8fd3827c277eb9d4a9b3709289d/blocks/large-promo-block/features/large-promo/default.jsx#L64

on this case, the missing class was an issue with the position of the new label.